index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="logo"><img src="./images/logo.png" alt="Logo"></div>
+        <nav class="myNav">
+         <a href="#"><img src="./images/hamburger.png" alt="hamburger" class="hamburger"></a>
+        </nav>
```

Comment:
- [ ] Great job using `<nav>` tag here 💯 However I think it would be better to wrap this part (header) with `<header>` tag, the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info 👍

For example
```
<header>
  <div class="logo"><img src="./images/logo.png" alt="Logo"></div>
    <nav class="myNav">
       <a href="#"><img src="./images/hamburger.png" alt="hamburger" class="hamburger"></a>
   </nav>
</header>
```